src/main/java/com/training/springpractice/controllers/CatalogController.java

```diff
@@ -0,0 +1,64 @@
+package com.training.springpractice.controllers;
+
+import com.training.springpractice.models.Catalog;
+import com.training.springpractice.models.CatalogBody;
+import com.training.springpractice.services.CatalogService;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.PropertyMap;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/api/v1/catalog-entry")
```

Comment:
Since you defined the path for this in this way your path will look like:  
  POST /api/vi/catalog-entry
If you will need a GET for obtain a single catalog data the path will be something like:
 GET /api/v1/catalog-entry/{id}
is that fine? 
As a good practice we need to define the Resources as nouns, so maybe use just **.../catalogs** should be enough

> URIs as resources as nouns
> One of the most recognizable characteristics of REST is the predominant use of nouns in URIs. RESTful URIs should not indicate any kind of CRUD (Create, Read, Update, Delete) functionality. Instead, REST APIs should allow you to manipulate a resource — which should take the form of a noun — through one of the main HTTP methods.
> Example: /users/{id} instead of /getUser